tape/src/main/java/com/squareup/tape/QueueFile.java

```diff
@@ -342,7 +342,7 @@ private int usedBytes() {
   }
 
   /** Returns number of unused bytes. */
-  private int remainingBytes() {
+  synchronized int remainingBytes() {
```

Comment:
Wouldn't it be enough that `usedBytes()` is `synchronized` and this is only a `read` method?